components/content-types/Form/FormWidget.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Form from './Form';
+const dotCMSApi = require('../../../utils/dotcms/dotcmsApi');
+
+function getFormVariableFromContent(renderedContent) {
+    return renderedContent.match(/variable='[^']+('*)/g)[0].split("'")[1];
+}
+
+const FormWidget = ({ formId, rendered }) => {
+    const [items, setItems] = useState([]);
+    let variable;
+
+    if (rendered) {
+        variable = getFormVariableFromContent(rendered);
+    }
+
+    useEffect(() => {
+        dotCMSApi
+            .form({ identifier: formId })
+            .get()
+            .then(({ fields }) => {
```

Comment:
Here we can get the variable name:

![Screen Shot 2019-12-10 at 10 40 21](https://user-images.githubusercontent.com/751424/70549269-849a1e00-1b39-11ea-89d0-795b5601b03a.png)